erpnext/accounts/doctype/asset/asset.py

```diff
@@ -80,7 +80,7 @@ def validate_asset_values(self):
 			if cint(self.number_of_depreciations_booked) > cint(self.total_number_of_depreciations):
 				frappe.throw(_("Number of Depreciations Booked cannot be greater than Total Number of Depreciations"))
 						
-		if self.next_depreciation_date and getdate(self.next_depreciation_date) < getdate(nowdate()):
+		if self.next_depreciation_date and getdate(self.next_depreciation_date) < getdate(self.purchase_date):
```

Comment:
Yes I got that warning. But why the next depreciation is validated against nowdate? I guess should be validated against purchase date.

Please advice.